ale_linters/fortran/fortitude.vim

```diff
@@ -0,0 +1,59 @@
+" Author: gomfol12
+" Desciption: A linter for fortran using fortitude.
+
+call ale#Set('fortran_fortitude_executable', 'fortitude')
+call ale#Set('fortran_fortitude_options', '')
+
+let s:severity_map = {
+\   'E': 'E',
+\   'C': 'W',
+\   'OB': 'I',
+\   'MOD': 'I',
+\   'S': 'I',
+\   'PORT': 'I',
+\   'FORT': 'I',
+\}
+
+function! ale_linters#fortran#fortitude#Handle(buffer, lines) abort
+    let l:json_str = join(a:lines, "\n")
+
+    try
+        let l:errors = json_decode(l:json_str)
+    catch
+        return []
+    endtry
+
+    if empty(l:errors)
+        return []
+    endif
+
+    let l:output = []
+
+    for l:error in l:errors
```

Comment:
We have a helper function for decoding JSON potentially split over many lines and returning a default value you can try here.

```suggestion
    let l:output = []

    for l:error in ale#util#FuzzyJSONDecode(a:lines, [])
```